consensus/ethash/ethash.go

```diff
@@ -320,15 +320,25 @@ func MakeDataset(block uint64, dir string) {
 	d.release()
 }
 
+// EthashConfig are the configuration parameters of the ethash.
+type EthashConfig struct {
+	CacheDir       string
+	CachesInMem    int
+	CachesOnDisk   int
+	DatasetDir     string
+	DatasetsInMem  int
+	DatasetsOnDisk int
+
+	// Miscellaneous options
+	PowFake   bool
+	PowTest   bool
+	PowShared bool
+}
```

Comment:
Please name this struct simply `Config`. We're in the `ethash` package, so your naming would stutter `ethash.EthashConfig`.